library/build.gradle

```diff
@@ -8,6 +19,7 @@ android {
         minSdkVersion 17
         targetSdkVersion 23
         versionCode 1
+        // NOTE: When updating versionName, be sure to update version in publishing group below too
```

Comment:
ugh, so I added the comment below right after I made this PR. However, I forgot to hit "standalone comment" and instead started a review. Please refer to the now-published comment.